### DISCUSSION TITLE
Dockerfile: disable npm update warnings

### DISCUSSION THREAD

**Discussion by adminradio** [directus/directus#12531] [Feature Requests]
Hi,
since updates are useless in epheremal containers and such a _warning_ should not catch any focus in logs (docker-compose, cloudwatch, ec2-tasks etc.), the `Dockerfile` can implement something like `RUN echo "update-notifier=false" >> ~/.npmrc` right after `USER node`.

I know, it's just cosmetic - but as an admin I can't overlook any _warning_ and it catches me every time. :-/

**Comment by rijkvanzanten** [comment]
Sounds good to me! Wanna create a PR to add that in?

**Reply by adminradio** [reply]
Sure, thank you.
[docker - disable npm update warnings #12547](https://github.com/directus/directus/pull/12547)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
